src/test/java/seedu/recipe/logic/commands/ExitCommandTest.java

```diff
@@ -9,12 +10,17 @@
 import seedu.recipe.model.ModelManager;
 
 public class ExitCommandTest {
-    private Model model = new ModelManager();
-    private Model expectedModel = new ModelManager();
+    private final Model model = new ModelManager();
+    private final Model expectedModel = new ModelManager();
 
     @Test
     public void execute_exit_success() {
         CommandResult expectedCommandResult = new CommandResult(MESSAGE_EXIT_ACKNOWLEDGEMENT, false, true);
         assertCommandSuccess(new ExitCommand(), model, expectedCommandResult, expectedModel);
     }
+
+    @Test
+    public void execute_nullModel() {
```

Comment:
may want to specify expected behaviour (NPE thrown)

Reply:
fixed